src/build.rs

```diff
@@ -67,7 +67,7 @@ pub fn cargo_build_wasm(
     step: &Step,
     extra_options: &Vec<String>,
 ) -> Result<(), Error> {
-    let msg = format!("{}Compiling to WASM...", emoji::CYCLONE);
+    let msg = format!("{}Compiling to Wasm...", emoji::CYCLONE);
```

Comment:
this feels like the weirdest of WASM, wasm, Wasm- any objection to making it all lower case?

Reply:
It's what the spec and official website use:

> WebAssembly (abbreviated Wasm) ...

> The Wasm stack machine is designed to be ...

Makes sense to me since it is still a proper noun